x-pack/plugins/monitoring/server/lib/beats/get_beat_summary.js

```diff
@@ -61,6 +65,8 @@ export async function getBeatSummary(req, beatsIndexPattern, { clusterUuid, beat
       'hits.hits._source.beats_stats.metrics.libbeat.output.write.bytes',
       'hits.hits._source.beats_stats.metrics.libbeat.config.reloads',
       'hits.hits._source.beats_stats.metrics.beat.info.uptime.ms',
+      'hits.hits._source.beats_stats.metrics.beat.handles.limit.hard',
```

Comment:
I couldn't find these fields in the data report, it looks more like:

```
...
                "harvester" : {
                  "open_files" : 34,
                  "skipped" : 0,
                  "started" : 34,
                  "closed" : 0,
                  "running" : 34
...
```

I think I miss something here.

Reply:
I'm confused. [This indicates it should be standardized](https://github.com/elastic/stack-monitoring/issues/3#issuecomment-427405579)?

Reply:
I think we cleared up this confusion. @ruflin had been testing on a OSX machine but that OS does not report file handle stats (whereas Linux and Windows do). I updated the PR description to include testing steps that mention this.